### DISCUSSION TITLE
Config help with `"groupName": "{{manager}}"`

### DISCUSSION THREAD

**Discussion by discentem** [renovatebot/renovate#27492] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

Is it possible to use `"groupName": "{{manager}}"` but then exclude specify dependencies? For example, I want to group all bazel dependencies into a single PR except one particular dependency? I'm looking for "group all except X" functionality essentially. 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Yes set matchManager and excludePackageNames

**Reply by discentem** [reply]
Thanks for the reply @rarkins! Will this cause a separate PR to be opened for that one excluded dependency? I don't want to exclude the dependency just have it updated alone through discrete PRs.

**Reply by rarkins** [reply]
Yes because that's the default behavior if you don't set a group

**Reply by discentem** [reply]
I tried this but it doesn't have the desired effect.

https://github.com/discentem/cavorite/blob/cb96a47645546f73fd77af25f314ef69832f31f5/renovate.json

```jsonc
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended"
  ],
  "packageRules": [
    {
      "matchManagers": ["bazel", "bazelisk", "github-actions", "gomod", "dockerfile"],
      "groupName": "{{manager}}",
      "excludePackageNames": ["rules_proto_grpc", "com_google_protobuf", "googleapis"]
    },
    {
      "matchManagers": ["bazel", "bazelisk", "github-actions", "gomod", "dockerfile"],
      "groupName": "{{manager}}", # <--- I thought this might open a PR for all the remain packages, the ones that were excluded above but didn't work
    }
  ],
  "commitMessagePrefix": "[Renovatebot]"
}
```

**Reply by discentem** [reply]
But this worked: 

https://github.com/discentem/cavorite/commit/217f313501320d77320a31a4dffc5708965908c0

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended"
  ],
  "packageRules": [
    {
      "matchManagers": ["bazel", "bazelisk", "github-actions", "gomod", "dockerfile"],
      "groupName": "{{manager}}",
      "excludePackageNames": ["rules_proto_grpc", "com_google_protobuf", "googleapis"]
    }
  ],
  "commitMessagePrefix": "[Renovatebot]"
}
```
